app/config/idplist-default.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "elevatedRights": true,
```

Comment:
wonder if `level: <int>` would work better (?)

Reply:
We don't really have multi-tiered levels of access so I don't think we should go that route yet; a boolean suffices.